packages/web-components/src/components/rux-clock/rux-clock.tsx

```diff
@@ -119,17 +131,66 @@ export class RuxClock {
         )
     }
 
+    private _handleDateIn() {
+        if (!this.dateIn!.includes('-')) {
```

Comment:
I'm not sure if this is the best way to determine if it's a unix date or not. Any better suggestions?